composer.json

```diff
@@ -9,8 +9,8 @@
         }
     ],
     "require": {
-        "php": "^7.2",
-        "php-translation/common": "^3.0",
+        "php": "^7.2 || ^8.0",
+        "php-translation/common": "^3.0 || ^4.0",
```

Comment:
I think it's still too early for allowing 4.0, we will tag v3.1.0 first, see https://github.com/php-translation/common/pull/43